src/app/admin/pages/learning-objects/learning-objects.component.ts

```diff
@@ -31,24 +31,26 @@ export class LearningObjectsComponent implements OnInit, OnDestroy {
   componentDestroyed$: Subject<void> = new Subject();
 
   ngOnInit(): void {
-    this.route.queryParams
-      .pipe(
-        takeUntil(this.componentDestroyed$),
-      )
-      .subscribe(u => this.getUserLearningObjects(u.username));
+    this.route.queryParams.subscribe(params => {
+      const username = params['username'];
+      if (typeof(username) !== 'undefined' && username !== null) {
```

Comment:
Can we be sure to scrub the typeof undefined statements from this PR? I think we had a conversation or two about this before. 
```
if (username == undefined) { ... }
```
does the same thing and is much less verbose.

Reply:
@DonSeannelly Is `if (username == undefined) { ... }` or `if (username == null) { ... }` the better option for this check?

Reply:
@gshaw1997 I think it is entirely personal opinion. Both are bottom values so the same operation is done under the hood.

Personally, I favor the if-null approach since null is fewer characters to type. I think that we should just pick one as a team and stick with it.